### PR TITLE
Prepare RT for create-react-app

### PR DESCRIPTION
**Pull request by bhavesh-desai-scratch**
## Purpose 
Maintaining Reactive traders build is becoming cumbersome. We should leverage the excellent work done by create-react-app and delete most our build. This change will make it easier to keep our tools up to date and have the support of the opensource community as time passes.

### Key changes

- Make sure all code is under source
- We are going to stop using sass and CSS pre-processors shortly when we reskin the application. 
- Create-react-app advocates compiling SASS separately to the main build. 
- To make things easier, I have moved all the SCSS files to a single folder
- We will be replacing these styles with a JS in CSS solution shortly

### CODE REVIEW

File: src/client/package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@types/openfin": "29.0.0",
```
Comment (fdore) [inline src/client/package.json]: would move to devDependencies for consistency

File: src/client/src/system/openFin/openFin.ts [hunk 2]
```diff
@@ -47,7 +46,7 @@ export default class OpenFin {
           currentWindow.restore(() => currentWindow.bringToFront(
             () => log.info('Window brought to front.'),
             err => log.error(err),
-          ), err => log.error(err))
+          ), () => log.error())
```
Comment (fdore) [inline src/client/src/system/openFin/openFin.ts]: why did you remove the error?

File: src/client/src/ui/styles/scss/blotter.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
-@import '../spotTile/variables';
-@import './variables';
+@import 'variables';
+@import 'variables';
```
Comment (fdore) [inline src/client/src/ui/styles/scss/blotter.scss]: duplicate import

**Comment by tdeheurles** [comment]
Nothing for Devops, fine for me

File: src/client/src/system/openFin/openFin.ts (hunk 2 above)
Comment (bhavesh-desai-scratch) [reply src/client/src/system/openFin/openFin.ts]: Does not exist according to TS definitions

Comment (bhavesh-desai-scratch) [reply src/client/src/system/openFin/openFin.ts]: Fixed

Comment (fdore) [reply src/client/src/system/openFin/openFin.ts]: It does according to the docs http://cdn.openfin.co/jsdocs/stable/fin.desktop.Window.html#restore
